merlin/models/tf/models/ranking.py

```diff
@@ -152,3 +160,68 @@ def DCNModel(
 
 def YoutubeDNNRankingModel(schema: Schema) -> Model:
     raise NotImplementedError()
+
+
+def DeepFMModel(
+    schema: Schema,
+    embedding_dim: int,
+    deep_block: Optional[Block] = MLPBlock([64, 128]),
+    prediction_tasks: Optional[
+        Union[PredictionTask, List[PredictionTask], ParallelPredictionBlock]
+    ] = None,
+    **kwargs
+):
+    """DeepFM-model architecture.
+
+    Example Usage::
+        depp_fm = DeepFMModel(schema, embedding_dim=64, deep_block=MLPBlock([256, 64]))
+        depp_fm.compile(optimizer="adam")
+        depp_fm.fit(train_data, epochs=10)
+
+    References
+    ----------
+    [1] Huifeng, Guo, et al.
+        "DeepFM: A Factorization-Machine based Neural Network for CTR Prediction"
+        arXiv:1703.04247  (2017).
+
+    Parameters
+    ----------
+    schema : Schema
+        The `Schema` with the input features
+    embedding_dim : int
+        Dimension of the embeddings
+    deep_block : Block
+        The `Block` that learns high-ordeer feature interactions
+    prediction_tasks: optional
+        The prediction tasks to be used, by default this will be inferred from the Schema.
+
+    Returns
+    -------
+    Model
+
+    """
+    num_features = len(schema.column_names)
+    cardinalities = schema_utils.categorical_cardinalities(schema)
+
+    input_block = InputBlock(
+        schema, embedding_options=EmbeddingOptions(embedding_dim_default=embedding_dim), **kwargs
+    )
+
+    pairwise_block = TabularBlock(StackFeatures(axis=-1)).connect(FMPairwiseInteraction())
+    deep_block = TabularBlock(ConcatFeatures()).connect(deep_block)
+    first_order_block = CategoricalOneHot(cardinalities).connect(
+        [
```

Comment:
`connect` can take in multiple arguments ([source](https://github.com/NVIDIA-Merlin/models/blob/main/merlin/models/tf/blocks/core/base.py#L353)), so `[` and `]` can be removed here